iped-api/src/main/java/iped/search/IMultiSearchResult.java

```diff
@@ -14,11 +15,16 @@
 public interface IMultiSearchResult {
 
     IItemId getItem(int i);
+    IIPEDSource getIPEDSource();
 
     Iterable<IItemId> getIterator();
 
     int getLength();
 
     float getScore(int i);
+    
+    /* returns true if the resultset includes the docid */
+    boolean hasDocId(int docId);
```

Comment:
I know iped-api already depends on some Lucene classes/info, but it shouldn't and I think we should not increase this kind of dependency. I don't remember why this was included, couldn't the issue you faced be fixed another way?

Reply:
This is important. Is with this method that I match if the doc is in the result (via internal bitset) to count it on the correspondent date.

Reply:
If I remember well, I have talked with you about this additional dependency and you said it could be done.

Reply:
Sorry, I don't remember about this specific method just about IIPEDSource dependency, this one is ok. I'll remove it from IMultiSearchResult interface and keep it in MultiSearchResult implementation.